Nix build fails on nixpkgs master: Tex error when generating manual

Steps to reproduce:

* Get latest nixpkgs (9d92bd)
* `nix-build release.nix -A binaryTarball.x86_64-linux`

```
 Ok [48] [49] [50] [51] [52] [53] [54] [55] [56] [57])
(/nix/store/qad2qwzfr28g5af33nnxvap4nf1gyq0r-tetex-3.0/share/texmf-dist/fonts/s
ource/jknappen/ec/exrligtb.mf
 Ok) ) ) )
(some charht values had to be adjusted by as much as 0.06586pt)
Font metrics written on ecrm1000.tfm.
Output written on ecrm1000.600gf (256 characters, 51364 bytes).
Transcript written on ecrm1000.log.
mktextfm: /tmp/nix-build-nix-tarball-1.10.drv-0/texfonts/tfm/jknappen/ec/ecrm1000.tfm: successfully generated.
pdflatex failed
/nix/store/108r62gg5bcnn18rnk7dq7ljiffa8g0z-dblatex-0.3.7/share/dblatex/latex/style/dbk_hyper.sty: File `bookmark.sty' not found.
/nix/store/108r62gg5bcnn18rnk7dq7ljiffa8g0z-dblatex-0.3.7/share/dblatex/latex/style/dbk_hyper.sty:33: Emergency stop.
/nix/store/108r62gg5bcnn18rnk7dq7ljiffa8g0z-dblatex-0.3.7/share/dblatex/latex/style/dbk_hyper.sty:33: leading text: }
Unexpected error occured
Error: pdflatex compilation failed
doc/manual/local.mk:83: recipe for target 'doc/manual/manual.pdf' failed
make: *** [doc/manual/manual.pdf] Error 1
build time elapsed:  0m0.110s 0m0.064s 0m18.051s 0m1.590s
builder for ‘/nix/store/m4kw5b6s9h186rifq5ky8zngljaidls1-nix-tarball-1.10.drv’ failed with exit code 2
```

--- a/release.nix
+++ b/release.nix
@@ -24,7 +24,7 @@ let
 
         buildInputs =
           [ curl bison flex perl libxml2 libxslt bzip2
-            tetex dblatex nukeReferences pkgconfig sqlite libsodium
+            dblatex (dblatex.tex or tetex) nukeReferences pkgconfig sqlite libsodium
             docbook5 docbook5_xsl
           ] ++ lib.optional (!lib.inNixShell) git;
 

